README.md

```diff
@@ -35,7 +35,38 @@ variable `LOG_LEVEL`.
 ### Exceptions
 
 All uncaught exceptions are automatically logged to `stderr` and the process is
-exited. There is no need to add an additional handler.
+exited. There is no need to add an additional handler. If you would like to
+enable the additional behavior of writing all exceptions to a file, you can!
+
+```
+const Logger = require('@modulus/logger');
+
+Logger.writeExceptions(PATH);
+
+// you can still use the factory to get a logger instance for this file
+var logger = Logger('namespace');
+logger.info('message', { example: true });
+```
+
+Note, you must use the required module directly, and create a logger instance
+separately. You typically only need to do this in your main file, so the
```

Comment:
Do you mean that you typically only need to create the logger instance separately in the main file?

So in the for example, in `index.js` you might have:
```
const Logger = require('@modulus/logger');
Logger.writeExceptions(PATH);

var logger = Logger('namespace');
logger.info('message');
```

while in another file you might have:
```
const Logger = require('@modulus/logger')('namespace');
Logger.info('message');
```